examples/example-select2-multiselect-editor.html

```diff
@@ -108,11 +108,11 @@ <h2>View Source:</h2>
   CC: 'Cocos (Keeling) Islands',
   CO: 'Colombia',
   KM: 'Comoros',
-  CG: 'Congo�(Brazzaville)',
+  CG: 'Congo�(Brazzaville)',
```

Comment:
Not sure how these got here

Reply:
yeah, not sure either. 😅

Reply:
There could have been "special character" whitespace rather than the real deal. Happens when people copy and paste things could have been there the whole time and no one ever woulda known lol 😂

Reply:
haha yeah that could be a notepad++ and/or copy & paste thing, feel free to fix it, those are just demos anyway. There's lot of surprises and goodies you can find on a 10 years old lib 😉